app/viewmodels/CheckYourAnswersHelper.scala

```diff
@@ -27,9 +27,18 @@ import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.Actions
 class CheckYourAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messages) extends SummaryListRowHelper {
 
   def rows(fileRole: FileRole): Seq[SummaryListRow] = {
-    Seq(
-      historicDatesRow(userAnswers.get(HistoricDateRequestPage(fileRole)), fileRole)
-    ).flatten
+    Seq(historicDatesRow(userAnswers.get(HistoricDateRequestPage(fileRole)), fileRole)).flatten
+  }
+
+  def dateRows(fileRole: FileRole): Option[String] = {
+    val res: Option[HistoricDates] = userAnswers.get(HistoricDateRequestPage(fileRole))
+
+    res.map { historicDates =>
+      HtmlFormat.escape(
+        messages("date.range",
+            Formatters.dateAsMonthAndYear(historicDates.start),
```

Comment:
Seems line 39 and 40 are not properly formatted (extra space before the start of the each line). If it shows correctly at your machine then leave it.